src/spend.rs

```diff
@@ -575,6 +576,8 @@ pub struct CreateSpendRes {
 /// * `main_descriptor`: the multipath Liana descriptor, used to derive the addresses of the
 /// candidate coins.
 /// * `secp`: necessary to derive data from the descriptor.
+/// * `use_primary_path`: whether the primary spending path should be used when estimating
+/// satisfaction size.
 /// * `tx_getter`: an interface to get the wallet transaction for the prevouts of the transaction.
 /// Wouldn't be necessary if we only spent Taproot coins.
```

Comment:
```suggestion
/// Wouldn't be necessary if we only spent non-taproot coins.
```
Not related to this PR but i notice this few weeks ago

Reply:
No, the original sentence is correct. :)

Reply:
my bad, i misunderstood when you told me about that! :man_facepalming: